resources/assets/js/app/views/topics/topicHeader.js

```diff
@@ -0,0 +1,115 @@
+var Marionette = require('backbone.marionette');
+var _ = require('underscore');
+var Radio = require('backbone.radio');
+var Bookmark = require('../../models/BookmarkModel');
+var currentUser = require('../../initializers/currentUser');
+
+module.exports = Marionette.ItemView.extend({
+    template: 'topicHeader',
+
+    modelEvents: {
+        'change': 'render'
+    },
+
+    serializeData: function () {
+        var meta = this.model.getMeta();
+        var id = this.model.get('id');
+
+        if (!meta) return {};
+
+        return {
+            model: this.model.toJSON(),
+            meta: {
+                user: meta.user,
+                likes: meta.likes,
+                comments: meta.comments,
+            }
+        };
```

Comment:
It would be better to return the object with the same properties if no meta:
return {
            model: {},
            meta: {}
}